js/script.js

```diff
@@ -0,0 +1,89 @@
+import {
+  $taskForm,
+  $taskTitle,
+  $taskList,
+  $btnCreate,
+} from "../js/elements.js";
+import { stripSanitizedParts } from "../js/utils/stripSanitizedParts.js";
+import { formatDate } from "../js/helpers/formatDate.js"; // Import the formatDate function
```

Comment:
Why are we using this comment?

Reply:
For reducing the possibility of confusion.

Reply:
Then why didn't we added to the other import statements?

Reply:
Because it was the latest modification and for better understanding for coding that specific part and not to get lost inside the code base and was meant to be removed later. Comments are generally removed before pushing but here it was overlooked.